src/app/shared/helpers/date-time.ts

```diff
@@ -9,6 +9,8 @@ export enum Duration {
   SECOND = 'second',
 }
 
+export type DateRange = 'today' | 'week' | 'fortnight' | '90 days' | '1 year' | '18 months';
```

Comment:
enum values please so they can match up with the isDuring function AND also be used in the dropdown list